### PR TITLE
Removed all autopanning/zooming + Meta-data for layers API Call .

### LINKED ISSUE #110: Bounding box ordering, re-centering issues?
I opened the Wisconsin Non-Metal layer but then it tries to recenter on a very broad bounding box:

image

https://publiclab.org/people#12/43.9929/-90.3883/BL1-Wisconsin_NM

Wondering if there is some better ordering we need to do to manage how the re-zooming leaves things once all layers are loaded. Ideas?

### LINKED ISSUE #208: Lagging of map in full screen due to large parsing of data
## Description
In some layers, (like osm ) layer loading takes a lot of time (looks like the browser is hanging ) due to huge data being parsed.
A quick fix would be to allow layer loading only at zoom levels by indicating zoom-in in brackets along with the layer name. 
But a more stable fix is necessary.
 
cc- @jywarren

### PR DESCRIPTION
**Pull request by sagarpreet-chadha**
Fixes  #110!
Fixes #208!

So only Wisconsin layer automatically zooms and pans the map on loading . 
So removed that feature .
Also minor bumped the version .

AND meta-data file created where we can give the information of each layer and when to call API (bounding box , min/max zoom , etc.) .

### CODE REVIEW

File: src/toxicReleaseLayer.js [hunk 1]
```diff
@@ -49,14 +46,27 @@ L.LayerGroup.ToxicReleaseLayer = L.LayerGroup.extend(
...
                 (function() {
                     var script = document.createElement("SCRIPT");
                     script.src = 'https://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js';
                     script.type = 'text/javascript';
                     var zoom = self._map.getZoom(), origin = self._map.getCenter() ;
+                    var extents = info.toxicReleaseLayer.extents;
+                    if(zoom < extents.minZoom){
+                      return;
+                    }
+                    var bounds = new L.LatLngBounds(
```
Comment (sagarpreet-chadha) [inline src/toxicReleaseLayer.js]: This basically allows the API call only when we are in USA .

**Comment by sagarpreet-chadha** [comment]
@jywarren ...how does this looks?

File: example/oneLinerCodeExample.html [hunk 2]
```diff
@@ -64,7 +64,7 @@
           ).addTo(map);
 
       L.LayerGroup.EnvironmentalLayers({
-        include: ['mapknitter', 'clouds'],
+        //include: ['mapknitter', 'clouds'],
```
Comment (jywarren) [inline example/oneLinerCodeExample.html]: Oops was this intentional?

File: src/info.json [hunk 3]
```diff
@@ -0,0 +1,137 @@
+{
+   "wisconsin": {
+      "name": "Wisconsin Non-Metallic Mining",
+      "url": "https://services.arcgis.com",
+      "api_url": "https://services.arcgis.com/jDGuO8tYggdCCnUJ/arcgis/rest/services/Nonmetallic_and_Potential_frac_sand_mine_proposals_in_West_Central_Wisconsin/FeatureServer/0/",
+      "extents": {
+        "NE_lat":  44.08758502824518,
```
Comment (jywarren) [inline src/info.json]: This is looking awesome. Great work. Are there maybe standard parameter formats from the Leaflet API for bounds? 

https://leafletjs.com/reference-1.4.0.html#bounds

This seems to show:

```js
[
    [40.712, -74.227],
    [40.774, -74.125]
]
```

...noting that each pair can also be a `LatLng` object.

File: src/layercode.js [hunk 4]
```diff
@@ -65,39 +65,43 @@ L.LayerGroup.LayerCode = L.LayerGroup.extend(
...
+           var info = require("./info.json");
                 (function() {
                     var zoom;
                     var Layer_URL;
                     var $ = window.jQuery;
 
-                    if (self.layer == "fractracker"){
-                        Layer_URL = "https://spreadsheets.google.com/feeds/list/19j4AQmjWuELuzn1GIn0TFRcK42HjdHF_fsIa8jtM1yw/o4rmdye/public/values?alt=json" ;
+                    if (self.layer === "fractracker"){
+                        Layer_URL = info.fractracker.api_url; ;
```
Comment (jywarren) [inline src/layercode.js]: Love this!!!!

File: src/info.json (hunk 3 above)
Comment (sagarpreet-chadha) [reply src/info.json]: Okay 👍

File: src/layercode.js (hunk 4 above)
Comment (sagarpreet-chadha) [reply src/layercode.js]: Thanks <3

File: example/oneLinerCodeExample.html (hunk 2 above)
Comment (sagarpreet-chadha) [reply example/oneLinerCodeExample.html]: Nope reverting this!

**Comment by jywarren** [comment]
Awesome!!!

**Pull request merged**